groups/data-lake/main.tf

```diff
@@ -14,6 +10,24 @@ data "aws_security_group" "mongo_db" {
   }
 }
 
+data "vault_generic_secret" "secrets" {
+  path = "applications/${var.aws_profile}/data-lake"
```

Comment:
Can introduce a `service` variable with a default value of `data-lake` to make this more consistent with other projects?

Reply:
It's also worth considering whether we will only ever need one compound configuration entry in Vault related to this service, or whether to create an additional level in the path to allow for related entries in future. We've adopted this approach in a few places, where the 'service' name is just another subdivision in the path, for example: https://github.com/companieshouse/sftp-gateway-terraform/blob/4ca90cbb4b5db5306e577cc7f8cf875ddc8a5d20/groups/sftp-gateway/sftp-gateway/transfer.tf#L2

Reply:
I agree this should use `service` as per other repositories. As for adding another layer within the hierarchy, as we've done to support environments elsewhere, I'm not sure it's required at the moment as I believe this is an account level project. Were we to introduce environment level config we'd need to revise other aspects of the code anyway. I'll add the service variable now...